NuGet/Microsoft.ChakraCore.Debugger.nuspec

```diff
@@ -1,14 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  Sample usage:
+  nuget.exe pack Microsoft.ChakraCore.Debugger.nuspec -Properties "id=some_id" -Version "some_version"
+-->
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
-    <id>Microsoft.ChakraCore.Debugger</id>
+    <id>$id$</id>
```

Comment:
Does this change it to "ChakraCore.Debugger" for the NuGet id?

Reply:
It allows setting the ID to whatever we find convenient.
If, for instance, in the future we deploy a NuGet called `Microsoft.ChakraDebugger`, we can do that without changing the NUSPEC.

The `id` can be injected via the `Properties` argument in the NuGet command.